src/helper/selection-tools/bounding-box-tool.js

```diff
@@ -105,27 +92,92 @@ class BoundingBoxTool {
             doubleClicked: doubleClicked
         };
         if (this.mode === BoundingBoxModes.MOVE) {
+            this.setCursor('grabbing');
             this._modeMap[this.mode].onMouseDown(hitProperties);
             this.removeBoundsHandles();
         } else if (this.mode === BoundingBoxModes.SCALE) {
             this._modeMap[this.mode].onMouseDown(hitResult, this.boundsPath, getSelectedRootItems());
             this.removeBoundsHandles();
         } else if (this.mode === BoundingBoxModes.ROTATE) {
+            this.setCursor('grabbing');
             this._modeMap[this.mode].onMouseDown(hitResult, this.boundsPath, getSelectedRootItems());
             // While transforming, don't show bounds
             this.removeBoundsPath();
         }
 
         return true;
     }
+    onMouseMove (event, hitOptions) {
+        this._updateCursor(event, hitOptions);
+    }
+    _updateCursor (event, hitOptions) {
+        const { mode, hitResult } = this._determineMode(event, false, hitOptions);
+        if (hitResult) {
+            if (mode === BoundingBoxModes.MOVE) {
+                this.setCursor('grab');
+            } else if (mode === BoundingBoxModes.ROTATE) {
+                this.setCursor('grab');
+            } else if (mode === BoundingBoxModes.SCALE) {
+                this.setSelectionBounds();
+                if (this.boundsPath.position.x === hitResult.item.position.x) {
```

Comment:
If you wanted to simplify this part, another way to get the paper.Point of the corners is
boundsPath.topLeft, topRight, bottomRight, bottomLeft
There's also leftCenter, topCenter, rightCenter, bottomCenter
http://paperjs.org/reference/rectangle/

You could also add more helper data to the circles themselves if that would help; take a look at line 283.

Reply:
Thanks for the tip! Fixed in 2216ac22f79f061ecd071292d689efb02148cd14.